src/GraphQL/Utilities/SchemaPrinterOptions.cs

```diff
@@ -24,5 +26,11 @@ public class SchemaPrinterOptions
         /// Indicates whether to print descriptions as comments for compatibility with the 2016 GraphQL specification.
         /// </summary>
         public bool PrintDescriptionsAsComments { get; set; } = true;
+
+        /// <summary>
+        /// Provides the ability to order the schema elements upon printing.
```

Comment:
Lets change comment for `ISchemaComparer`:
>  Provides the ability to order the schema elements upon introspection.
->
>  Provides the ability to order the schema elements upon introspection or printing.

Reply:
Done.